hbw/app/assets/stylesheets/hbw/brand_variables.sass

```diff
@@ -22,6 +22,7 @@
   //colors
   //box-shadow + border + background
   --base-hbw-light-gray: #ECF0F1
+  --base-hbw-description-gray: #BDBDBD
   --base-hbw-attacher-color: #BDBDBD
   --base-hbw-claimimg-popup-description-color: #BDBDBD
   //background + color in inputs and table
```

Comment:
We have at least four base colours with the same values, maybe we should aggregate it in some way.